impeller/docs/standalone_gles.md

```diff
@@ -0,0 +1,195 @@
+# Using Impeller as a Standalone Rendering Library (with OpenGL ES)
+
+This guide describes how to use Impeller as a standalone rendering library using OpenGL ES. Additionally, some form of WSI (Window System Integration) is essential. Since EGL is the most the most popular form of performing WSI on platforms with OpenGL ES, Impeller has a toolkit that assists in working with EGL. This guide will use that toolkit.
```

Comment:
trivial nit (feel free to ignore): when introducing a new acronym, it's generally clearer to use the form `bla bla Apple Banana Cherry (ABC)` rather than `bla bla ABC (Apple Banana Cherry)`.

Reply:
Done.